src/main/java/io/appium/java_client/AppiumDriver.java

```diff
@@ -202,28 +208,42 @@ protected static Capabilities substituteMobilePlatform(Capabilities originalCapa
         }
     }
 
-    @Override public Set<String> getContextHandles() {
+    @Override
+    public Set<String> getContextHandles() {
         Response response = execute(DriverCommand.GET_CONTEXT_HANDLES);
         Object value = response.getValue();
         try {
             List<String> returnedValues = (List<String>) value;
             return new LinkedHashSet<>(returnedValues);
         } catch (ClassCastException ex) {
             throw new WebDriverException(
-                "Returned value cannot be converted to List<String>: " + value, ex);
+                    "Returned value cannot be converted to List<String>: " + value, ex);
         }
     }
 
-    @Override public String getContext() {
+    @Override
+    public String getContext() {
         String contextName =
-            String.valueOf(execute(DriverCommand.GET_CURRENT_CONTEXT_HANDLE).getValue());
+                String.valueOf(execute(DriverCommand.GET_CURRENT_CONTEXT_HANDLE).getValue());
         if ("null".equalsIgnoreCase(contextName)) {
             return null;
         }
         return contextName;
     }
 
-    @Override public DeviceRotation rotation() {
+    /**
+     * This method is used to get build version status of running Appium server.
+     *
+     * @return map containing version details
+     */
+    public Map<String, String> getStatus() {
+        Map<String, Object> statusMap = (Map<String, Object>) execute(DriverCommand.STATUS).getValue();
+
+        return (Map<String, String>) statusMap.get("build");
```

Comment:
why cannot we just return the statusMap ?

Reply:
Also, this is not a very good idea to cast `Map<String, Object>` to `Map<String, String>`

Reply:
@mykola-mokhnach For simplicity, it has been implemented this way.
@jlipps , @saikrishna321 , @SrinivasanTarget What do you guys think? Does it need further changes or can be merged?
If we really want to enhance it further, I believe the better approach will to have pojo classes created for JSON response (i.e. {"build":{"version":"1.13.0"}}, in this case 2 pojo classes will be created one parent pojo containing an instance of Build class and getter/setter for it and another pojo file "Build" containing a field version and getter/setter for that) and then we do the parsing for extracting version field. Please suggest.

Reply:
We don't need to add more POJO's here. We just need not do the casting and send the status map as is. POJO is overkill to me. Heavy lifting should be done by users as per their needs.

Reply:
@SrinivasanTarget 
You mean getStatus() should look like this:

**public Map<String, Object> getStatus() {
        Map<String, Object> statusMap = (Map<String, Object>) execute(DriverCommand.STATUS).getValue();
        return statusMap;
    }**